front/app/api/event_attendance/useEventAttendances.ts

```diff
@@ -0,0 +1,28 @@
+import { useQuery } from '@tanstack/react-query';
+import { CLErrors } from 'typings';
+import fetcher from 'utils/cl-react-query/fetcher';
+import { isNilOrError } from 'utils/helperUtils';
+import { IEventAttendances, EventAttendanceKeys } from './types';
+import eventsAttendancesKeys from './keys';
+
+const fetchEventAttendances = ({ eventId }: { eventId: string }) => {
```

Comment:
Minor. Since it is only one, we could use
```suggestion
const fetchEventAttendances = (eventId: string) => {
```